CMakeLists.txt

```diff
@@ -57,6 +57,10 @@ find_package(Pango)
 find_package(GdkPixbuf)
 find_package(PAM)
 
+find_package(LibInput REQUIRED)
+add_definitions(${LIBINPUT_DEFINTIONS})
+include_directories(${LIBINPUT_INCLUDE_DIRS})
```

Comment:
Can you put the include/add_definitions only where they're required, i.e. here https://github.com/SirCmpwn/sway/blob/master/sway/CMakeLists.txt#L1-L7